pkgs/development/python-modules/django-crispy-forms/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv
+, fetchPypi
+, buildPythonPackage
+}:
+buildPythonPackage rec {
+
+  pname = "django-crispy-forms";
+  version = "1.7.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "5952bab971110d0b86c278132dae0aa095beee8f723e625c3d3fa28888f1675f";
+  };
+
+  # Checks are designed to be run by module's authors not users. Requires
```

Comment:
This could be said about most of the test suites we use. Doesn't stop us, because enabling tests makes the package collection far more robust.

Reply:
How should I proceed with resolving this?
The testing procedures that they have written come with instructions for creating a django app before testing. I could create a repo that contains that skeleton and pull it potentially as an additional source.

Reply:
Yeah it's quite likely it's not feasible to get them going, it was just an offhand remark tbh.

Reply:
Word.